helm_deploy/README.md

```diff
@@ -1,19 +1,19 @@
 
 ### Example deploy command
 ```
-helm --namespace offender-categorisation-dev  --tiller-namespace offender-categorisation-dev upgrade offender-categorisation ./offender-categorisation/ --install --values=values-dev.yaml --values=example-secrets.yaml
+helm --namespace offender-categorisation-dev  --tiller-namespace offender-categorisation-dev upgrade offender-risk-profiler ./offender-risk-profiler/ --install --values=values-dev.yaml --values=example-secrets.yaml
```

Comment:
worth doing a global search and replace for `categorisation` with `risk-profiler`, still lots of occurrences that I presume should all be changed

Reply:
In this case this is correct.  The offender-risk-profiler app lives inside the `offender-categorisation-dev` namespace.